crates/net/network-api/src/lib.rs

```diff
@@ -49,6 +49,9 @@ pub trait NetworkInfo: Send + Sync {
 
     /// Returns `true` if the network is undergoing sync.
     fn is_syncing(&self) -> bool;
+
+    /// Returns `true` when a fresh node is initially being synced, known as a Pipeline sync
```

Comment:
this is not quite accurate because this also returns true on restart if the first pipeline sync is active.

Reply:
because during a pipeline sync, state of the sync "job" is stored externally I think, and it'll be loaded in/continued from upon restart